Source/WebCore/html/HTMLScriptElement.cpp

```diff
@@ -86,10 +87,39 @@ void HTMLScriptElement::didFinishInsertingNode()
     ScriptElement::didFinishInsertingNode();
 }
 
+void HTMLScriptElement::setTextExported(String&& value)
```

Comment:
The call site can just write `IMPL->setText(String(...))` and then you don't need to name this "exported". The conversion from `NSString` does not **need** to be implicit.